tests/httpwrappers/tests.py

```diff
@@ -699,6 +703,42 @@ def test_repeated_nonstandard_keys(self):
         """
         self.assertIn('good_cookie', parse_cookie('a:=b; a:=c; good_cookie=yes').keys())
 
+    def test_python_cookies(self):
+        # from Python's Lib/test/test_http_cookies.py
+        self.assertEqual(parse_cookie('chips=ahoy; vienna=finger'), {'chips': 'ahoy', 'vienna': 'finger'})
+
+        # here we differ from how python parses cookies
+        self.assertEqual(
+            parse_cookie('keebler="E=mc2; L=\\"Loves\\"; fudge=\\012;"'),
+            {'keebler': '"E=mc2', 'L': '\\"Loves\\"', 'fudge': '\\012', '': '"'})
+
+        # Check illegal cookies that have an '=' char in an unquoted value
```

Comment:
Use consistent comment style. Here we have "Check" that doesn't appear in the comments below. In this test you have a blank line between each comment and test, but not in the garbage test.